feat(article): VueJS conference Amsterdam 2022

tbusser-io opened the pull request:
These are my takeaways of the VueJS conference in Amsterdam 2022. It
is what I wrote for the Vue Slack channel augmented with some extra
information which colleagues posted in response to my article.

In this PR you will find:
- My author information
- My avatar
- The article with the takeaways of the conference

Commit:
feat(article): VueJS conference Amsterdam 2022

These are my takeaways of the VueJS conference in Amsterdam 2022. It
is what I wrote for the Vue Slack channel augmented with some extra
information which colleagues posted in response to my article.

In this PR you will find:
- My author information
- My avatar
- The article with the takeaways of the conference

sanderdejong88 commented on data/blog/vue-js-conference-amsterdam-2022.md:
```diff
@@ -0,0 +1,63 @@
+authors: ['thijs-busser']
+---
+
+# VueJS Conference Amsterdam June 2 & 3
```
Can you remove this title, it's already added to the page based on the frontmatter.

sanderdejong88 commented on data/blog/vue-js-conference-amsterdam-2022.md:
```diff
@@ -0,0 +1,63 @@
+
+# VueJS Conference Amsterdam June 2 & 3
+
+Together with some colleagues I’ve attended the VueJS conference in Amsterdam. Here are some of the takeaways from these two days.
```
I would move this to the `summary` in the frontmatter. That way it's above the image

tbusser-io committed:
Update article

- Added a group photo at the top of the article
- Rewrote the intro so the article has a better setup
- Changed the heading levels

tbusser-io committed:
Update the summary

Moved the first paragraph to the summary.
